api/rs/slint/mcu.md

```diff
@@ -13,36 +13,26 @@ Writing an application in Rust that runs on a MCU requires several prerequisites
 * Install tools for flashing and debugging your code on the device.
 
 We recommend reading the [Rust Embedded Book](https://docs.rust-embedded.org/book/),
-as well as the curated list of [Awesome Embedded Rust](https://github.com/rust-embedded/awesome-embedded-rust) for a wide range of different
-crates, tools and training materials. These resources should guide you through the initial setup and often come with "hello world" examples
+and the curated list of [Awesome Embedded Rust](https://github.com/rust-embedded/awesome-embedded-rust) for a wide range of
+crates, tools and training materials. These resources should guide you through the initial setup. Many include a "hello world" example
 to get started with your device.
 
-Slint requires a global memory allocator. That is currently only possible in the nightly version of Rust, because the support for using a custom global
-allocator in a bare metal environment with `#![no_std]` has not been stabilized yet (see [#51540](https://github.com/rust-lang/rust/issues/51540) or
-[#66741](https://github.com/rust-lang/rust/issues/66741) for tracking issues).
+Slint requires a custom global memory allocator in a bare metal environment with `#![no_std]`.
+The rust team hasn't yet stabilized this feature, so only the nightly version
```

Comment:
Note that this is stabilized in the upcomming 1.68   (one of the tracking issue is closed)